drivers/at86rf215/at86rf215.c

```diff
@@ -115,13 +116,13 @@ void at86rf215_reset(at86rf215_t *dev)
     }
 
     /* disable clock output */
-#if AT86RF215_USE_CLOCK_OUTPUT == 0
+#if !IS_ACTIVE(CONFIG_AT86RF215_USE_CLOCK_OUTPUT)
```

Comment:
This can be turned into a C conditional, using `IS_ACTIVE`.

Reply:
Okay. Shall help make code more readable I guess

Reply:
Yes, and let's the compiler check the code. See https://www.kernel.org/doc/html/v4.10/process/coding-style.html#conditional-compilation

Reply:
Thanks for letting me know

Reply:
Please change it to a C conditional